docs/machine-learning/automate-training-with-model-builder.md

```diff
@@ -13,11 +13,17 @@ Model Builder uses automated machine learning (AutoML) to explore different mach
 
 You don't need machine learning expertise to use Model Builder. All you need is some data, and a problem to solve. Model Builder generates the code to add the model to your .NET application.
 
-![Model Builder Visual Studio extension user interface animation](media/ml-dotnet-model-builder.gif)
+![Model Builder Scenarios](./media/model-builder-scenarios.png)
```

Comment:
@briacht what do you think about the screenshot? Should we zoom out a bit to show MB is part of Visual Studio?

Reply:
Yeah I think showing all of Visual Studio would be good